dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -485,6 +544,18 @@ public static boolean startTransactionIfNeeded() throws DotDataException {
         return startTransaction;
     }
 
+    public static void commit () throws DotDataException {
+
+        try {
+            if (inTransaction()) {
+                DbConnectionFactory.getConnection().commit();
+
+            }
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)